fusestore/store/src/meta_service/raftmeta.rs

```diff
@@ -127,27 +127,13 @@ pub struct MetaStore {
 // }
 
 impl MetaStore {
-    /// Create a new `MetaStore` instance.
-    #[tracing::instrument(level = "info")]
-    pub async fn new(id: NodeId, config: &configs::Config) -> common_exception::Result<MetaStore> {
-        // TODO: move id into config.
-        let mut config = config.clone();
-        config.id = id;
-
-        let (ms, _is_open) = Self::open_create(&config, None, Some(())).await?;
-        Ok(ms)
-    }
-
-    /// Open an existent `MetaStore` instance.
-    pub async fn open(config: &configs::Config) -> common_exception::Result<MetaStore> {
-        let (ms, _is_open) = Self::open_create(config, Some(()), None).await?;
-        Ok(ms)
-    }
-
     /// Open an existent `MetaStore` instance or create an new one:
     /// 1. If `open` is `Some`, try to open an existent one.
     /// 2. If `create` is `Some`, try to create one.
     /// Otherwise it panic
+    /// Returns MetaStore instance and a bool indicating if it is opened from a previous state.
+    ///
+    /// TODO(xp): make the is_open flag a field of MetaStore, maybe introduce a trait to define this behavior
     #[tracing::instrument(level = "info")]
     pub async fn open_create(
```

Comment:
What do you think about renaming it to `open`

Reply:
`open_create` is fine since meta_node maybe not exists and needs to boot.

Reply:
`open()` feels like just-loading-from-disk or something similar.
This function tries to open an existent one or to create a new one from nothing, depending on the two args `open` and `create`.

It's meant to remind readers that this function could be used to open, to create, or to create-if-can-not-open.
`Something::open()` being used to create something may confuse readers. 🤔

Reply:
ok